tools/docker/stop.sh

```diff
@@ -43,13 +43,18 @@ main() {
         esac
     done
 
-    while read -r container; do
-        docker "$stop_cmd" "${container}" >/dev/null 2>&1
-        if [ "$remove" = true ] ; then
-            docker rm "${container}" >/dev/null 2>&1
-        fi
-    done <"$containers_file"
-
+    grep -v '^network ' "$containers_file" |\
+        while read -r container; do
+            docker "$stop_cmd" "${container}" >/dev/null 2>&1 || true
+            if [ "$remove" = true ] ; then
+                docker rm "${container}" >/dev/null 2>&1 || true
+            fi
+        done
+
+    sed -n '/^network \(.*\)$/s//\1/p' "$containers_file" |\
+        while read -r network; do
+            docker network rm -f "${network}" >/dev/null 2>&1 || true
```

Comment:
The `-f` option doesn't seem to be available (at least not in  19.03.4) for `network rm`.
It should also check for "$remove". I don't know what happens if you try to remove a network used by some containers, I guess it just fails.

Reply:
Well, that's exactly what the -f option does: remove any containers connected to that network. But apparently it's podman-specific, then.